core/services/job/models.go

```diff
@@ -359,17 +359,14 @@ type OCR2OracleSpec struct {
 }
 
 func (s *OCR2OracleSpec) RelayIdentifier() (relay.Identifier, error) {
```

Comment:
WDYT about s/Identifier/ID? :shrug: 
```suggestion
RelayID uniquely identifies a relay.
func (s *OCR2OracleSpec) RelayID() (relay.Identifier, error) {
```

Reply:
done